conda_smithy/configure_feedstock.py

```diff
@@ -102,7 +102,13 @@ def warn_once(msg: str):
 
 @cache
 def rattler_build_version() -> str | None:
-    p = subprocess.run(["rattler-build", "--version"], text=True, capture_output=True)
+    if sys.platform == "win32":
+        path = os.path.join(sys.prefix, "Library", "bin")
+    else:
+        path = os.path.join(sys.prefix, "bin")
+    p = subprocess.run(
+        [f"{path}/rattler-build", "--version"], text=True, capture_output=True
+    )
```

Comment:
This is incorrect for [current rattler-build](https://conda-metadata-app.streamlit.app/?q=conda-forge%2Fwin-64%2Frattler-build-0.46.0-h18a1a76_0.conda), which puts `rattler-build.exe` directly into `%CONDA_PREFIX%\bin`, not under `\Library\bin`.